### ISSUE TITLE
Each request creates a conversation

### ISSUE THREAD

**Issue by 17604515707** [Soulter/hugging-chat-api#31]
Each request will create a dialogue, each generated chat_id how to reuse in the next request? I did not find the relevant api instructions

**Comment by Soulter** [comment]
I just tested it and there's no problem.
![image](https://github.com/Soulter/hugging-chat-api/assets/37870767/1d9b4e44-507d-4b79-89ca-b06a527b31b2)

Perhaps you create hugchat.ChatBot repeatedly?

**Comment by 17604515707** [comment]
> I just tested it and there's no problem. ![image](https://user-images.githubusercontent.com/37870767/241692756-1d9b4e44-507d-4b79-89ca-b06a527b31b2.png)
> 
> Perhaps you create hugchat.ChatBot repeatedly?

Yes, the chatid generated by the request in one run is unchanged, I mean reuse the previous chatid.
Example: The chatid of the first request is 1234. For the second request, I want to use the chatid of 1234 instead of regenerating a new chatid, which can ensure the coherence of the chat information

**Comment by RockChinQ** [comment]
> > I just tested it and there's no problem. ![image](https://user-images.githubusercontent.com/37870767/241692756-1d9b4e44-507d-4b79-89ca-b06a527b31b2.png)
> > Perhaps you create hugchat.ChatBot repeatedly?
> 
> Yes, the chatid generated by the request in one run is unchanged, I mean reuse the previous chatid. Example: The chatid of the first request is 1234. For the second request, I want to use the chatid of 1234 instead of regenerating a new chatid, which can ensure the coherence of the chat information

see #5 , tracking this feature there.
